jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -139,7 +135,7 @@ public void set(String image) {
 
     @Parameter private Map<String, String> labels = Collections.emptyMap();
 
-    @Parameter private String appRoot = JavaLayerConfigurations.DEFAULT_APP_ROOT;
+    @Nullable @Parameter private String appRoot;
```

Comment:
I remember the Gradle counterpart set this to an empty string (`""`) instead of null. Can we be consistent? is there some reason they cannot be consistent? Not sure whether an empty string or null is better though.

Reply:
Ok